tensorflow_model_optimization/python/core/sparsity_tf2/sparse_utils.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Convenience functions for sparse training."""
+
+import tensorflow as tf
+
+class Bernouilli(tf.keras.initializers.Initializer):
+  """
+  Initialization distributio following a Bernouilli process..
+  """
+
+  def __init__(self, p):
+    """
+    p: probability parameter of success (i.e. 1).
```

Comment:
Make this an explicit "args" section, have the description from above in here as well